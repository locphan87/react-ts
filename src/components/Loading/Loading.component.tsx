import { Intent, Spinner } from '@blueprintjs/core'
import React from 'react'

const Loading = () => (
  <div className="flex items-center justify-center h-screen">
    <Spinner intent={Intent.PRIMARY} size={50} />
  </div>
)

export default Loading
