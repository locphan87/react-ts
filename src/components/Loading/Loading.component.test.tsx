import { testSnapshots } from '../../utils/test.util'
import Loading from './Loading.component'

describe('render', () => {
  testSnapshots(Loading, [
    {
      description: 'render loading'
    }
  ])
})
