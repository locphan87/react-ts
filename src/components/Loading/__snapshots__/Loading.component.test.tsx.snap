// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render snapshots render loading 1`] = `
<div>
  Loading...
</div>
`;
